test/Flow_02_test/MC_workflow.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+libdir = os.path.join(os.path.split(
+         os.path.dirname(os.path.realpath(__file__)))[0],"C:\\Users\\Klara\\Documents\\Intec\\MLMC\\src\\mlmc")
+sys.path.insert(1,libdir)
+
+from gmsh_io import GmshIO, ElementType
+from operator import add
+import numpy as np
+from  correlated_field import SpatialCorrelatedField
+import matplotlib.pyplot as plt
+import re
+from scipy.stats.mstats import mquantiles
+import seaborn as sns
+
+# Read the mesh network of the model:
+gio = GmshIO()

```

Comment:
Introduce a class for Flow calculation with random fields.
Methods:
- __init__(self, yaml_file_abs_path, mesh_rel_path, value_mesh_path_rel)
- add_field(name, spatial_correlated_field, mu, sigma)
- _extract_results(output_dir)
- run() ... evaluate fields, write value_mesh, run flow, extract results